src/FsCodec.Box/TryDeflate.fs

```diff
@@ -22,10 +30,18 @@ module private EncodedMaybeDeflated =
         output.ToArray()
     let decode struct (encoding, data) : ReadOnlyMemory<byte> =
         if encoding = int Encoding.Deflate then inflate data |> ReadOnlyMemory
```

Comment:
could match int encoding  with 1/2 and call a single fn that selects the decompressor

Reply:
You can use Enum values in matches right? Take a look now